### PR TITLE
MAJOR: Updated tests to pass & Travis support

### PR DESCRIPTION
**Pull request by shoaibali**
Github no longer has an ID on search box
Avatar class has been applied to commits as well
Assumes non-logged in user
Example now also shows how to use partialLinkText.

### CODE REVIEW

**Comment by DavertMik** [comment]
Why did you close? 0.o

**Comment by shoaibali** [comment]
@DavertMik I figured probably not worthy of your kind consideration?

**Comment by DavertMik** [comment]
Not, I'm just a slowpoke sometimes )
Yes, I noticed that change in GitHub design in my other project which uses it as a test base, but it's hard to follow all my projects (

**Comment by DavertMik** [comment]
Thanks for this patch
